### PR TITLE
Update Blog Guidelines

### PR DESCRIPTION
**Pull request by pablovem**
Initial [Guidelines](https://github.com/angular-medellin/meetup/wiki/Blog-Guidelines) have been added to the wiki.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Sólo asegúrate de seguir [**estas instrucciones**](./assets/SPEAKERS.md), noso
 - 💡 Nos encantan las [**propuestas de mejora**](https://github.com/angular-medellin/meetup/issues?q=is%3Aopen+is%3Aissue+label%3A%22Propuesta+de+mejora%22), nos mantienen con la mente abierta
 - 💼 Nuestros miembros son muy activos. [**Aquí**](https://github.com/angular-medellin/meetup/wiki) puedes encontrar los proyectos en los que actualmente estamos trabajando.
 - 🎤 Seguro hay un tema que te mueres por aprender ¿Qué charla te gustaría escuchar [**la próxima vez**](https://github.com/angular-medellin/meetup/issues?q=is%3Aopen+is%3Aissue+label%3A%22Pr%C3%B3ximo+tema%22)?
-- :page_facing_up: También puedes publicar un artículo en nuestro [blog](https://medium.com/angularmedellin), puedes encontras los pasos a seguir [*aquí*](https://github.com/angular-medellin/meetup/wiki).
+- :page_facing_up: Tambien puedes publicar un articulo en nuestro [blog](https://medium.com/angularmedellin), puedes encontras los pasos a seguir [**aquí**](https://github.com/angular-medellin/meetup/wiki/Blog-Guidelines).
```
Comment (jdjuan) [inline CONTRIBUTING.md]: Quitaste las tildes! 
raisins_face

File: README.md [hunk 2]
```diff
@@ -84,7 +84,7 @@ Sólo asegúrate de seguir [**estas instrucciones**](./assets/SPEAKERS.md), noso
 - 💡 Nos encantan las [**propuestas de mejora**](https://github.com/angular-medellin/meetup/issues?q=is%3Aopen+is%3Aissue+label%3A%22Propuesta+de+mejora%22), nos mantienen con la mente abierta
 - 💼 Nuestros miembros son muy activos. [**Aquí**](https://github.com/angular-medellin/meetup/wiki) puedes encontrar los proyectos en los que actualmente estamos trabajando.
 - 🎤 Seguro hay un tema que te mueres por aprender ¿Qué charla te gustaría escuchar [**la próxima vez**](https://github.com/angular-medellin/meetup/issues?q=is%3Aopen+is%3Aissue+label%3A%22Pr%C3%B3ximo+tema%22)?
-- :page_facing_up: También puedes publicar un artículo en nuestro [blog](https://medium.com/angularmedellin), puedes encontras los pasos a seguir [*aquí*](https://github.com/angular-medellin/meetup/wiki).
+- :page_facing_up: Tambien puedes publicar un articulo en nuestro [blog](https://medium.com/angularmedellin), puedes encontras los pasos a seguir [**aquí**](https://github.com/angular-medellin/meetup/wiki/Blog-Guidelines).
```
Comment (jdjuan) [inline README.md]: Aquí también 😛

File: CONTRIBUTING.md (hunk 1 above)
Comment (pablovem) [reply CONTRIBUTING.md]: 😱 !!!

**Comment by pablovem** [comment]
😱  Sorry about that @jdjuan , [It's my first day](https://www.youtube.com/watch?v=9ZlOhSt_qW0).

**Pull request merged**